packages/integration-tests/api.js

```diff
@@ -309,6 +309,52 @@ async function getExecutionStatus({ prefix, arn }) {
   return JSON.parse(payload.body);
 }
 
+/**
+ * Add a collection to Cumulus via the API
+ *
+ * @param {Object} params - params
+ * @param {string} params.prefix - the prefix configured for the stack
+ * @param {string} params.collection - a collection object
+ * @returns {Promise<Object>} - the POST confirmation from the API
+ */
+async function addCollection({ prefix, collection }) {
+  const payload = await callCumulusApi({
+    prefix: prefix,
+    functionName: 'ApiCollectionsDefault',
+    payload: {
+      httpMethod: 'POST',
+      resource: '/collections',
+      path: '/collections',
+      body: JSON.stringify(collection)
+    }
+  });
+
+  return payload;
```

Comment:
This separate return isn't necessary, you could just return the Promise created on line 321.  The same goes for the `addProvider()` function.